apps/commudle-admin/src/app/feature-modules/notifications/components/community-notifications/community-notifications.component.html

```diff
@@ -1,5 +1,5 @@
-<button nbButton size="small" (click)="unread()" status="primary">Unread</button>
-<button nbButton size="small" (click)="allNotifications()" status="primary">All</button>
+<button nbButton size="small" (click)="unread()" status="primary" class="unread">Unread</button>
+<button nbButton size="small" (click)="allNotifications()" status="primary" class="allButton">All</button>
```

Comment:
classes name should be all-notifications 
unread-notifications

Reply:
Done